templates/web/admin/editusers.html.twig

```diff
@@ -1099,6 +1108,36 @@
         });
       }
     });
+
+    $('.select-all-bdd-right').click(function () {
+      $('.main-right-item > div').removeClass('unchecked').addClass('checked');
+      $('.main-right-item > div > input').val(1);
+    });
+
+    $('.select-all-bdd').click(function () {
+      $(this).closest('.right-items').find('div').show().removeClass('unchecked').addClass('checked');
+      $(this).closest('.right-items').find('input').val(1);
+      $(this).closest('.right-items').find('.unselect-all-bdd').removeClass('hide');
+      $(this).addClass('hide');
+    });
+    $('.unselect-all-bdd').click(function () {
+      $(this).closest('.right-items').find('div').hide().addClass('unchecked').remove('checked');
+      $(this).closest('.right-items').find('.case_right_access div').show();
+      $(this).closest('.right-items').find('input').val(0);
+      $(this).closest('.right-items').find('.select-all-bdd').removeClass('hide');
+      $(this).addClass('hide');
+    });
+
+   function countCheckedRight() {
+     $(".right-items" ).each(function() {
+       let count_checked = $( this ).find('input.checked').length;
+       if ( count_checked == 17) {
```

Comment:
better to compare number of ckboxes (N) with number of checked (n1) and/or unchecked (n0)